librarian_questions.py

```diff
@@ -0,0 +1,20 @@
+import fetchProcedures
+import start_questions

```

Comment:
I'm not sure you need these imports! I didn't see either modules being used in this file